### ISSUE TITLE
Keyboard tool bar issue

### ISSUE THREAD

**Issue by abhishekCO2** [MortenGregersen/KeyboardBarTest#1]
I can replicate similar issue as mentioned in video in Xcode 15.1 with iOS 17.3.1 iPhone 13 Device.
Do we have any solution for this?

**Comment by MortenGregersen** [comment]
No, I don’t have a solution. I used a DTS, but Apple had no workaround…

**Comment by abhishekCO2** [comment]
ohh okay and what's the DTS 😅?

**Comment by bartcone** [comment]
https://gist.github.com/JamesSedlacek/2d0425319e2a854da8c51f4b05c9842a

**Comment by nikolacirjakovic** [comment]
This can still be reproduced in 18.0 and 18.2.

**Comment by nikolacirjakovic** [comment]
![Image](https://github.com/user-attachments/assets/80f32ef6-1815-4ad8-9c55-e2bebdbeadb7)
It appears to be working on the iOS 26 beta.
